scrapy/utils/misc.py

```diff
@@ -176,7 +175,7 @@ def create_instance(objcls, settings, crawler, *args, **kwargs):
 def build_from_crawler(
     objcls: type[T], crawler: Crawler, /, *args: Any, **kwargs: Any
 ) -> T:
-    """Construct a class instance using its ``from_crawler`` constructor.
+    """Construct a class instance using its ``from_crawler`` or ``from_settings`` constructor.
```

Comment:
I wonder if we need this given `from_settings` is deprecated, but no strong opinion against it.